### ISSUE TITLE
git settings

### ISSUE THREAD

**Issue by adrianmiriuta** [conuthead/f051bldc#5]
You need to do some git cleanup and basic settings .
I cannot follow your changes.
You push lot of object , map ,listfiles.
add a .gitignore file in your project directory
and make a new begin (remove .git directory , git init )

[gitignore.zip](https://github.com/conuthead/f051bldc/files/3071676/gitignore.zip)

**Comment by AlkaMotors** [comment]
@adrianmiriuta 
Yes, I made a mistake last night when pushing changes. It was not intentional. I was working from another pc and really only wanted to write the changes to main.c.  I am planning to make some major changes to the structure. 
I am sorry it made it difficult to follow. Your testing on the wraith is helping the project along hugely right now!
